### PR TITLE
Add `profileMode` argument

### PR DESCRIPTION
**Pull request by spydon**
Adding the `profileMode` argument so that it adds the `--profile` flag to `flutter build`. (defaults to false)

### CODE REVIEW

**Comment by hey24sheep** [comment]
@spydon Hi, thank you for the PR. I have added a few notes.

**Comment by spydon** [comment]
> @spydon Hi, thank you for the PR. I have added a few notes. 

Hey! I think you forgot to submit the review,, because I can't see any comments?

**Review by hey24sheep — CHANGES_REQUESTED** [review]

File: tasks/build/index.js [hunk 1]
```diff
@@ -56,7 +57,7 @@ function main() {
             let targetPlatform = task.getInput('iosTargetPlatform', false);
             let codesign = task.getBoolInput('iosCodesign', false);
             let exportOptionsPlist = task.getInput('exportOptionsPlist', false);
-            yield buildIpa(flutterPath, targetPlatform == "simulator", codesign, buildName, buildNumber, debugMode, buildFlavour, entryPoint, dartDefine, dartDefineMulti, isVerbose, extraArgs, isIPA, exportOptionsPlist);
+            yield buildIpa(flutterPath, targetPlatform == "simulator", codesign, buildName, buildNumber, debugMode, profileMode, buildFlavour, entryPoint, dartDefine, dartDefineMulti, isVerbose, extraArgs, isIPA, exportOptionsPlist);
```
[the remark this answers is not in the document]
Comment (spydon) [reply tasks/build/index.js:60]: Done! Added it to web too.

**Comment by hey24sheep** [comment]
@spydon thanks for quick changes, but you made changes to .JS files, they get generated by .TS compiler. You need to edit these in .TS files also, `profileMode` has to be added to "task.json" so it can show up on UI of Azure Builds within this folder. Have a look [here](https://github.com/hey24sheep/azure-flutter-tasks/pull/50/files). you can refer to this PR for a guide.

**Comment by spydon** [comment]
@hey24sheep fixed!

**Review by hey24sheep — APPROVED** [review]

**Pull request merged**